app/routes/application.js

```diff
@@ -0,0 +1,35 @@
+import Route from '@ember/routing/route';
+import ENV from 'website-my/config/environment';
+
+const API_BASE_URL = ENV.BASE_API_URL;
+export default class ApplicationRoute extends Route {
+  model = async () => {
+    const defaultName = 'User';
+    const defaultPicture = 'dummyProfilePicture.png';
+
+    try {
+      const response = await fetch(`${API_BASE_URL}/users/self`, {
+        credentials: 'include',
+      });
+      const userData = await response.json();
+      if (response.status === 200 && !userData.incompleteUserDetails) {
+        const firstName = userData.first_name;
+        const profilePictureURL = userData.picture.url;
+        return (
+          { firstName, profilePictureURL } ?? { defaultName, defaultPicture }
+        );
```

Comment:
How about?
```suggestion
        const firstName = userData.first_name || defaultName;
        const profilePictureURL = userData.picture.url || defaultPicture;
        return { firstName, profilePictureURL };
```

Reply:
Ohh this is a great idea, nice suggestion will implement this